src/Makefile.sources

```diff
@@ -118,6 +118,8 @@ HB_BASE_sources = \
 	hb-ot-shape.cc \
 	hb-ot-shape.hh \
 	hb-ot-stat-table.hh \
+	hb-ot-style.cc \
+	hb-ot-style.h \
```

Comment:
.fon files (and I guess Type 1) could provide lots of provided things, so maybe -ot- removal?

Reply:
Also whether this should have a separated header file or not.